lib/src/transformer/impl_generation.dart

```diff
@@ -402,10 +402,6 @@ class ImplGenerator {
             annotations.Accessor accessorMeta = instantiateAnnotation(field, annotations.Accessor);
```

Comment:
We should also probably emit a warning if they try to use `@requiredProp`/`nullableRequiredProp`/`@Accessor` together.